cas-server-webapp/src/main/webapp/WEB-INF/webflow/login/login-webflow.xml

```diff
@@ -34,6 +34,7 @@
         <evaluate expression="clientAction" />
         <transition on="success" to="sendTicketGrantingTicket" />
         <transition on="error" to="ticketGrantingTicketCheck" />
+        <transition on="remote" to="remoteAuthenticate" />
```

Comment:
As mentioned before, the spring config for intercepted flow for institution login.